tests/core/engine_adapter/integration/config.yaml

```diff
@@ -89,6 +89,14 @@ gateways:
       cluster: cluster1
     state_connection:
       type: duckdb
+  inttest_risingwave:
```

Comment:
For the integration tests to run on CircleCI without the `port 5432 failed: server closed the connection unexpectedly` error, you'll also need to add a branch for risingwave to `.circleci/wait-for-db.sh` to block until the DB server is available

Reply:
Hi Erin, thanks for your help, I have updated and now there are some issues with the integration tests and I am trying to fix them.